css/elements.css

```diff
@@ -1425,3 +1439,28 @@ li.menu-search-result-term:before {
   background-color: #eee;
   box-shadow: inset 0 -1px 0 #ccc;
 }
+
+#metadata-block {
+  margin: 4em 0;
+  padding: 10px;
+  border: 1px solid #ee8421;
+}
+
+#metadata-block h1 {
+  font-size: 1.5em;
+  margin-top: 0;
+}
+#metadata-block > ul {
+  list-style-type: none;
+  margin: 0; padding: 0;
+}
+
+#ecma-logo {
+  width: 500px;
+}
+
+.unicode-property-table {
+  table-layout: fixed;
+  width: 100%;
+  font-size: 80%;
+}
```

Comment:
I thought we were going to move this into ecma262

Reply:
Wait why? That’s not specific to 262.

Reply:
Yes, this class only exists on a table in ecma262: https://github.com/tc39/ecma262/blob/815aa8a4d68ce24bec81ba6789c37d55bc87b54a/table-binary-unicode-properties.html#L3.

Reply:
`metadata-block` does appear [in 402](https://github.com/tc39/ecma402/blob/ef423a80538a83e395e63068210778562de1209a/spec/metablock.html#L2). I thought unicode-property-table did too, but I was mistaken. I’ll make that edit.